### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

**Comment by Pacane** [comment]
Why isn't json_annotation not in dev_dependencies? Is it required to be in regular deps?

File: tool/phases.dart [hunk 1]
```diff
@@ -2,8 +2,11 @@ import 'package:build_runner/build_runner.dart';
 import 'package:json_serializable/generators.dart';
 import 'package:source_gen/source_gen.dart';
 
-final PhaseGroup phases = new PhaseGroup.singleAction(
-    new PartBuilder(const [
-      const JsonSerializableGenerator(),
-    ]),
-    new InputSet('github', const ['lib/src/common.dart']));
+final buildActions = <BuildAction>[
```
Comment (Pacane) [inline tool/phases.dart]: shouldn't the file be renamed, or is this convention?

Comment (kevmoo) [reply tool/phases.dart]: Yes!

**Comment by kevmoo** [comment]
> Why isn't json_annotation not in dev_dependencies? Is it required to be in regular deps?

It's actually in the code that is shipped. **strictly** speaking, I don't think a missing import related to an annotation will affect runtime behavior –

**Comment by Pacane** [comment]
> It's actually in the code that is shipped. strictly speaking, I don't think a missing import related to an annotation will affect runtime behavior – 

Yeah it makes sense. I don't even know why I asked this lol. I guess I was tired.

**Pull request merged**